graphyte.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, host, port=2003, prefix=None, timeout=5, interval=None,
         Use "tags" to specify common or default tags for this Sender, which
         are sent with each metric along with any tags passed to send().
         """
+
```

Comment:
Don't worry for now, but it's best to avoid extraneous formatting in other repos, unless you're changing it to match the existing style in that repo. People have different styles and it just adds noise to the PR (and potential for bikeshedding).